deployment/BUILD

```diff
@@ -21,6 +23,22 @@ k8s_objects(
     ],
 )
 
+# Set --//deployment:image_tag to set the tag when running container_push
+# targets. If unset, defaults to "latest".
+#
+# Note that this tag is shared across all targets, which is fine since only one
+# container_push target can run at once.
+string_flag(
+    name = "image_tag",
+    build_setting_default = "latest",
```

Comment:
Should we set this to something like `dev`? The flag should generally be set and if something accidentally doesn't specify it, I don't think we should default to touching the latest tag.

Reply:
Done (used `development` instead of `dev` to avoid potential confusion with the dev environment)